src/main/charts/confluence/values.yaml

```diff
@@ -736,6 +736,57 @@ confluence:
   #
   additionalJvmArgs: []
 
+  # -- By default Tomcat's server.xml is generated in the container entrypoint from a template
+  # shipped with an official Confluence image. However, server.xml generation may fail if container
+  # is not run as root, which is a common case if Confluence is deployed to OpenShift.
+  #
+  tomcatConfig:
+
+    # -- Mount server.xml as a ConfigMap. Override configuration elements if necessary
+    #
+    generateByHelm: false
+
+    mgmtPort: "8005"
```

Comment:
nit: Is it intended that `mgmtPort` and `maxThreads` have different value from default ones?

Reply:
@yzha645 thanks for catching that! Pocy-paste from Jira. Fixed.